### PR TITLE
Enable recently read sorting option on ios

### PR DESCRIPTION
**Pull request by satindar**

### CODE REVIEW

File: apple/OmnivoreKit/Sources/Models/LinkedItemSort.swift [hunk 1]
```diff
@@ -30,23 +27,21 @@ public extension LinkedItemSort {
...
     switch self {
-    case .newest /* , .relevance */:
+    case .newest:
       return [NSSortDescriptor(keyPath: \LinkedItem.savedAt, ascending: false)]
     case .oldest:
       return [NSSortDescriptor(keyPath: \LinkedItem.savedAt, ascending: true)]
-//    case .recentlyRead:
-//      return [NSSortDescriptor(keyPath: \LinkedItem.updatedAt, ascending: false)]
+    case .recentlyRead:
+      return [NSSortDescriptor(keyPath: \LinkedItem.updatedAt, ascending: false)]
```
Comment (jacksonh) [inline apple/OmnivoreKit/Sources/Models/LinkedItemSort.swift:44]: I think you want `.readAt` here

Comment (satindar) [reply apple/OmnivoreKit/Sources/Models/LinkedItemSort.swift:44]: oh nice. didn't know we had that now 👍🏽

Comment (jacksonh) [reply apple/OmnivoreKit/Sources/Models/LinkedItemSort.swift:44]: @satindar oh, i think you have to add it 😬

Comment (jacksonh) [reply apple/OmnivoreKit/Sources/Models/LinkedItemSort.swift:44]: Sorry, thought you were fetching that and storing in LinkedItem already.

**Comment by satindar** [comment]
@jacksonh This one is using `readAt` now. Only weird thing now is that if you have the "recently read" sort activated and then tap an article in the list it will be moved to the top when you pop the view.

Maybe not that weird though.

File: apple/OmnivoreKit/Sources/Models/LinkedItemSort.swift [hunk 2]
```diff
@@ -30,23 +27,24 @@ public extension LinkedItemSort {
...
-    case .newest /* , .relevance */:
+    case .newest:
       return [NSSortDescriptor(keyPath: \LinkedItem.savedAt, ascending: false)]
     case .oldest:
       return [NSSortDescriptor(keyPath: \LinkedItem.savedAt, ascending: true)]
-//    case .recentlyRead:
-//      return [NSSortDescriptor(keyPath: \LinkedItem.updatedAt, ascending: false)]
+    case .recentlyRead:
+      return [
+        NSSortDescriptor(keyPath: \LinkedItem.readAt, ascending: false),
```
Comment (jacksonh) [inline apple/OmnivoreKit/Sources/Models/LinkedItemSort.swift:45]: nice, was wondering how we could do this.

**Review by jacksonh — APPROVED** [review]

**Pull request merged**